articles/privacy-art-collection.md

```diff
@@ -0,0 +1,35 @@
+|   |   |
+|---|---|
+| Title               | Privacy Art Collection |
+| Author              | OfficialCryptos |
+| Featured text       | FIXME |
```

Comment:
Need featured test and a better title?

Reply:
I like the title. but i'll suggest two more: 1) Veiled Visions: Art Collection on the Essence of Privacy; 2)The Art of Discretion: Exploring Privacy Through Creativity.

Reply:
I quit like the second "The Art of Discretion: Exploring Privacy Through Creativity."

Reply:
I like that second title!